pom.xml

```diff
@@ -391,6 +507,36 @@
         <scope>test</scope>
       </dependency>
 
+      <dependency>
+        <groupId>org.jglue.cdi-unit</groupId>
+        <artifactId>cdi-unit</artifactId>
+        <version>3.1.2</version>
+        <scope>test</scope>
+        <exclusions>
+          <exclusion>
```

Comment:
Do we still need to exclude this even we explicitly define a dependency down below?